README.md

```diff
@@ -154,6 +154,7 @@ Questions can be asked by raising an `Issue`.
 - [Evandro Rodrigues](https://github.com/evnrodr)
 - [essarrgee](https://github.com/essarrgee)
```

Comment:
I know it's not your responsibility but if you could move this line above by 1 place, it will be alphabetically correct. Just noticed hence suggested.

Reply:
Raise an issue - We can fix that in another PR

Reply:
I fixed it in #246